src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -147,18 +150,12 @@ public void setPerson(Person target, Person editedPerson) {
     public void associateOrdersWithPersons() {
         for (CustomerOrder customerOrder : customerOrderList.getOrders()) {
             addressBook.findPersonByPhone(customerOrder.getPerson().getPhone())
-                    .ifPresent(person -> {
-                        customerOrder.setOriginalPerson(person);
-                        person.addOrder(customerOrder);
-                    });
+                    .ifPresent(person -> person.addOrder(customerOrder));
```

Comment:
Think setoriginalperson should not be removed as it sets a pointer in the order object to the person object in addressbook during app initialization. Correct me if I am wrong

Reply:
I am not in charge of this check style code for this class. It should be alright I feel.